### ISSUE TITLE
Model not available - ConnectionAbortedError

### ISSUE THREAD

**Issue by Christoph-B** [rebane2001/matterport-dl#61]
Hello,
after several downloads and manual edits to add all the patchs, I don't manage to access to my model.
The download has no problem, I had added the JS files that I needed.

When I want to see the model, the log gives me the following error and I have the famous page "Oops, model not available...":
```
  File "C:\Users\XXXXXXXXXXXX\AppData\Local\Programs\Python\Python310\lib\socketserver.py", line 826, in write
    self._sock.sendall(b)
ConnectionAbortedError: [WinError 10053] Une connexion établie a été abandonnée par un logiciel de votre ordinateur hôte
```
I don't know if it's my python configuration or a new issue which is the problem.
I downloaded the favicon but, obvisouly, it didn't solve my issue.
I tried Firefox, Edge and Vivaldi but same result.

Here is more log if needed:
```
127.0.0.1 - - [19/Jun/2022 21:12:56] "GET / HTTP/1.1" 200 -
127.0.0.1 - - [19/Jun/2022 21:12:56] "GET /css/showcase.css HTTP/1.1" 200 -
127.0.0.1 - - [19/Jun/2022 21:12:56] "GET /css/unsupported_browser.css HTTP/1.1" 200 -
127.0.0.1 - - [19/Jun/2022 21:12:56] "GET /js/browser-check.js HTTP/1.1" 200 -
----------------------------------------
Exception occurred during processing of request from ('127.0.0.1', 52431)
Traceback (most recent call last):
  File "C:\Users\XXXXXXXXXXXX\AppData\Local\Programs\Python\Python310\lib\socketserver.py", line 316, in _handle_request_noblock
    self.process_request(request, client_address)
  File "C:\Users\XXXXXXXXXXXX\AppData\Local\Programs\Python\Python310\lib\socketserver.py", line 347, in process_request
    self.finish_request(request, client_address)
  File "C:\Users\XXXXXXXXXXXX\AppData\Local\Programs\Python\Python310\lib\socketserver.py", line 360, in finish_request
    self.RequestHandlerClass(request, client_address, self)
  File "C:\Users\XXXXXXXXXXXX\AppData\Local\Programs\Python\Python310\lib\http\server.py", line 651, in __init__
    super().__init__(*args, **kwargs)
  File "C:\Users\XXXXXXXXXXXX\AppData\Local\Programs\Python\Python310\lib\socketserver.py", line 747, in __init__
    self.handle()
  File "C:\Users\XXXXXXXXXXXX\AppData\Local\Programs\Python\Python310\lib\http\server.py", line 425, in handle
    self.handle_one_request()
  File "C:\Users\XXXXXXXXXXXX\AppData\Local\Programs\Python\Python310\lib\http\server.py", line 413, in handle_one_request
    method()
  File "C:\Users\XXXXXXXXXXXX\Downloads\matterport-dl-main_2022-05-29\matterport-dl.py", line 432, in do_GET
    SimpleHTTPRequestHandler.do_GET(self)
  File "C:\Users\XXXXXXXXXXXX\AppData\Local\Programs\Python\Python310\lib\http\server.py", line 658, in do_GET
    self.copyfile(f, self.wfile)
  File "C:\Users\XXXXXXXXXXXX\AppData\Local\Programs\Python\Python310\lib\http\server.py", line 858, in copyfile
    shutil.copyfileobj(source, outputfile)
  File "C:\Users\XXXXXXXXXXXX\AppData\Local\Programs\Python\Python310\lib\shutil.py", line 198, in copyfileobj
    fdst_write(buf)
  File "C:\Users\XXXXXXXXXXXX\AppData\Local\Programs\Python\Python310\lib\socketserver.py", line 826, in write
    self._sock.sendall(b)
ConnectionAbortedError: [WinError 10053] Une connexion établie a été abandonnée par un logiciel de votre ordinateur hôte
----------------------------------------
127.0.0.1 - - [19/Jun/2022 21:12:56] "GET /js/showcase.js HTTP/1.1" 200 -
----------------------------------------
Exception occurred during processing of request from ('127.0.0.1', 52432)
Traceback (most recent call last):
  File "C:\Users\XXXXXXXXXXXX\AppData\Local\Programs\Python\Python310\lib\socketserver.py", line 316, in _handle_request_noblock
    self.process_request(request, client_address)
  File "C:\Users\XXXXXXXXXXXX\AppData\Local\Programs\Python\Python310\lib\socketserver.py", line 347, in process_request
    self.finish_request(request, client_address)
  File "C:\Users\XXXXXXXXXXXX\AppData\Local\Programs\Python\Python310\lib\socketserver.py", line 360, in finish_request
    self.RequestHandlerClass(request, client_address, self)
  File "C:\Users\XXXXXXXXXXXX\AppData\Local\Programs\Python\Python310\lib\http\server.py", line 651, in __init__
    super().__init__(*args, **kwargs)
  File "C:\Users\XXXXXXXXXXXX\AppData\Local\Programs\Python\Python310\lib\socketserver.py", line 747, in __init__
    self.handle()
  File "C:\Users\XXXXXXXXXXXX\AppData\Local\Programs\Python\Python310\lib\http\server.py", line 425, in handle
    self.handle_one_request()
  File "C:\Users\XXXXXXXXXXXX\AppData\Local\Programs\Python\Python310\lib\http\server.py", line 413, in handle_one_request
    method()
  File "C:\Users\XXXXXXXXXXXX\Downloads\matterport-dl-main_2022-05-29\matterport-dl.py", line 432, in do_GET
    SimpleHTTPRequestHandler.do_GET(self)
  File "C:\Users\XXXXXXXXXXXX\AppData\Local\Programs\Python\Python310\lib\http\server.py", line 658, in do_GET
    self.copyfile(f, self.wfile)
  File "C:\Users\XXXXXXXXXXXX\AppData\Local\Programs\Python\Python310\lib\http\server.py", line 858, in copyfile
    shutil.copyfileobj(source, outputfile)
  File "C:\Users\XXXXXXXXXXXX\AppData\Local\Programs\Python\Python310\lib\shutil.py", line 198, in copyfileobj
    fdst_write(buf)
  File "C:\Users\XXXXXXXXXXXX\AppData\Local\Programs\Python\Python310\lib\socketserver.py", line 826, in write
    self._sock.sendall(b)
ConnectionAbortedError: [WinError 10053] Une connexion établie a été abandonnée par un logiciel de votre ordinateur hôte
----------------------------------------
127.0.0.1 - - [19/Jun/2022 21:12:56] "GET /?m=PtphgvSgYr4 HTTP/1.1" 200 -
127.0.0.1 - - [19/Jun/2022 21:12:57] "GET /js/browser-check.js HTTP/1.1" 200 -
127.0.0.1 - - [19/Jun/2022 21:12:57] "GET /js/showcase.js HTTP/1.1" 200 -
127.0.0.1 - - [19/Jun/2022 21:12:57] "GET /fonts/roboto-regular.woff2 HTTP/1.1" 200 -
127.0.0.1 - - [19/Jun/2022 21:12:57] "GET //api/v2/users/current HTTP/1.1" 200 -
127.0.0.1 - - [19/Jun/2022 21:12:57] code 404, message File not found
127.0.0.1 - - [19/Jun/2022 21:12:57] "GET /favicon.ico HTTP/1.1" 404 -
127.0.0.1 - - [19/Jun/2022 21:12:57] "GET /js/136.js HTTP/1.1" 200 -
127.0.0.1 - - [19/Jun/2022 21:12:57] "POST /api/mp/models/graph HTTP/1.1" 200 -
127.0.0.1 - - [19/Jun/2022 21:12:57] "GET /locale/messages/strings_fr.json HTTP/1.1" 200 -
127.0.0.1 - - [19/Jun/2022 21:12:57] "GET /locale/strings.json HTTP/1.1" 200 -
127.0.0.1 - - [19/Jun/2022 21:12:57] "GET /fonts/roboto-300.woff2 HTTP/1.1" 200 -
127.0.0.1 - - [19/Jun/2022 21:12:57] "GET /fonts/roboto-500.woff2 HTTP/1.1" 200 -
```
Do you have any idea ?

**Comment by mu-ramadan** [comment]
@Christoph-B  try this fix  #66

**Comment by mitchcapper** [comment]
This should be fixed now, please test again on master and let us know if not
